extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/media/GoogleMediaExporter.java

```diff
@@ -213,11 +214,15 @@ private ExportResult<MediaContainerResource> exportPhotosContainer(
     for (PhotoModel photo : container.getPhotos()) {
       GoogleMediaItem googleMediaItem =
           getGoogleMediaItem(photo.getIdempotentId(), photo.getDataId(), photo.getName(), authData);
+
       if (googleMediaItem == null) {
         continue;
       }
-
-      photosBuilder.add(GoogleMediaItem.convertToPhotoModel(Optional.empty(), googleMediaItem));
+      try {
+        photosBuilder.add(GoogleMediaItem.convertToPhotoModel(Optional.empty(), googleMediaItem));
+      } catch(ParseException e) {
+        // Note all errors are logged in GoogleMediaItem so no need to re-log them here.
```

Comment:
It is not logged in GoogleMediaItem, only an exception is thrown. In these catch blocks, I think we should log it and mention that the skipping happened here. do this for all callers of 
GoogleMediaItem.convertToPhotoModel

Reply:
Done.